website/docs/plugins/ax_sweeper.md

```diff
@@ -107,5 +109,6 @@ ax_config:
   client:
     verbose_logging: false
     random_seed: null
+  is_exact: false
```

Comment:
Probably not a big deal but being exact is not the same as being certain. Ax cares about the uncertainty measure so maybe we could call it `is_certain` (or something else).

Reply:
good point. maybe `is_errorless`?

Reply:
Is there a term that Ax docs/codebase uses that we can reuse?

Reply:
good point! changed to `is_errorless`.

Reply:
The parameter that is being set to 0 is called `SEM` which stands for "standard error of the mean" which can be truncated to "error" in this context, which is why I believe `is_errorless` fits.